compiler/rustc_driver_impl/src/lib.rs

```diff
@@ -667,7 +668,8 @@ fn print_crate_info(
                     return Compilation::Continue;
                 };
                 let t_outputs = rustc_interface::util::build_output_filenames(attrs, sess);
-                let crate_name = passes::get_crate_name(sess, attrs);
+                let crate_name = get_crate_name(sess, attrs);
```

Comment:
I believe all changes to this file can be reverted now. It only changes a `passes::get_crate_name` to `get_crate_name` + import of `passes::get_crate_name`.

Reply:
mhm, you're right